test/test.py

```diff
@@ -147,6 +148,14 @@ def test_request_review_incorrect_project_with_repo(self):
                                    repo_name=self.config['repo_name'])
         self.assertTrue('Project not found' in str(context.exception))
 
+class GerritTest(TestCase):
+    def setUp(self):
+        filename = join(dirname(__file__), 'test_gerrittest.yaml')
+        with open(filename, 'r') as f:
+            self.config = yaml.load(f)
+
+    def test_gerrit_object_create(self):
+        self.assertTrue(isinstance((get_git_service('gerrit')), GerritService))
```

Comment:
It would be good to have more test than just instantiation of the class. For example mocking a few API results for changes, mocking error responses etc.